src/chains/celo/fees.ts

```diff
@@ -0,0 +1,87 @@
+import type { Client } from '../../clients/createClient.js'
+import { type Address, type ChainFees, type Hex } from '../../index.js'
+
+import { formatters } from './formatters.js'
+
+export const fees: ChainFees<typeof formatters> = {
+  /*
+   * Estimate the fees per gas for a transaction
+   * If the transaction is to be paid in a token, the fees are estimated in the value of the token
+   * otherwise the default estimateFeesPerGas is used.
```

Comment:
I think i wrote this? but its wrong now. since  it will either return the fees or null to indicate that default should be estimations should be done